src/pages/guides/migrations/standalone_to_dx_experience_cloud_spa.md

```diff
@@ -0,0 +1,92 @@
+
+## Who is This Migration Guide For?
+
+If you are having trouble viewing your application in Adobe Experience Cloud, you may have initialized your application as a 
+Standalone Application, and this guide may help you. 
+
+Applications built with App Builder can only be accessed via the App Builder Catalog in Adobe Experience Cloud if they have been initialized as **DX 
+Experience Cloud SPA v1**. 
+
+## How can I tell if my project is a Standalone Application?
+
+Your application will **not** have the `App Builder ExcShell` sub-heading in Adobe Exchange. 
+
+### Standalone Application 
+<img width="806" alt="image" src="https://user-images.githubusercontent.com/28722775/173623479-3deebcc4-695a-4387-a714-b3eecc682c34.png">
+
+### DX Experience Cloud SPA v1
+![image](https://user-images.githubusercontent.com/28722775/173436294-9ecca357-1b63-441f-ad4a-b1cb903ebd1c.png)
+
+## Understanding the Difference 
+
+See [this](https://github.com/AdobeDocs/project-firefly/edit/main/src/pages/guides/extensions/extension_migration_guide.md#old-file-structure) guide to 
+better understand the difference between a Standalone Application and a DX Experience Cloud SPA v1. This guide also provides an overview of the
+directory structure for a DX Experience Cloud SPA v1 application (Under **New File Structure**) and a deeper dive into extension point configuration. 
+
+## Migration Steps 
+
+1. Modify local directory 
+    1. Create a new `src/dx-excshell-1` folder
+    2. Move `web-src`, `test`, `e2e`, `actions` under `src/dx-excshell-1`
+2. Add extension configuration 
+    1. Create `src/dx-excshell-1/ext.config.yaml` 
+    3. Copy the contents of `app.config.yaml` to `src/dx-excshell-1/ext.config.yaml` 
+    4. Delete the `application` tag in `src/dx-excshell-1/ext.config.yaml` 
+    5. Shift contents of `src/dx-excshell-1/ext.config.yaml` left one tab space 
+    6. Add an `operations` tag at the top of `src/dx-excshell-1/ext.config.yaml` with the following contents: 
+        ```
+        operations:
+          view:
+            - type: web
+              impl: index.html
+        ```
+    6. Replace the contents of `app.config.yaml` with the following: 
+        ```
+        extensions:
+            dx/excshell/1:
+                $include: src/dx-excshell-1/ext.config.yaml
+        ```
+3. Revoke the application (Optional: Only if application is published)
+    1. If admin
+        1. Navigate to Adobe Exchange 
+        2. Select **Manage** in the top toolbar 
+        3. Select **Approved Apps** in the top sub toolbar
+        4. Find your application in the list, click **Revoke**
+        5. Click **Revoke**
+    2. If not admin
+        1. Ping Admin to revoke
+        2. Wait for admin to revoke 
```

Comment:
The previous line capitalizes Admin, while it is lowercase here. Should be `Admin`?